eos/maths/multiplepolylog_li22.hh

```diff
@@ -0,0 +1,138 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+// This file is made by Hjalte Frellesvig, Damiano Tommasini, and Christopher Wever
+// For further details see ArXiv:1601.02649
+// Last modification: 28th Jan. 2016
+
+/*
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef EOS_GUARD_EOS_MATHS_MULTIPLEPOLYLOG_HH
+#define EOS_GUARD_EOS_MATHS_MULTIPLEPOLYLOG_HH 1
+
+#include <eos/maths/complex.hh>
+
+namespace eos
+{
+    complex<double> li22(const complex<double> & x, const complex<double> & y) __attribute__ ((pure));
+
+    namespace li22_impl
```

Comment:
Can you move this part of the header into a dedicated file ``multiplepolylog-li22-const.hh``?
Also, please make sure to rename all file from ``_li22`` to ``-li22``.